OpenEdXMobile/res/layout/activity_register.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/cl_root"
```

Comment:
need id for title on all screens

Reply:
following story has been created for that
https://openedx.atlassian.net/browse/LEARNER-4408